### ISSUE TITLE
Accessibility: Comment 'Remove' link not tabbable

### ISSUE THREAD

**Issue by HuongNV13** [frankkoch/moodle-mod_studentquiz#125]
Hi Frank,
We found an issue related to accessibility as below:

-----------------------------------------------------------------
The 'remove' link used to remove comments is not accessible for keyboard or screen reader users

Expected:

- Users can tab to 'Remove' and press 'Enter' button on the keyboard to remove the comment in the question

- Screen Reader reads 'Remove comment' when users tab to 'Remove'

Thanks,
